### PR TITLE
Adding scrolloffset property

### PR DESCRIPTION
**Pull request by malaniz**
Fix for this issue
https://github.com/fisshy/react-scroll/issues/143

### CODE REVIEW

**Comment by fisshy** [comment]
Thanks for the PR, Why not just increase the offset?

File: modules/mixins/Helpers.js [hunk 1]
```diff
@@ -141,7 +141,7 @@ var Helpers = {
 
 
 
-            var offsetY = y - this.props.offset;
+            var offsetY = y - this.props.offset + this.props.scrolloffset;
```
Comment (a-omsk) [inline modules/mixins/Helpers.js]: what about defaultProps?

**Comment by fisshy** [comment]
Could you write a test for this? And please make "ignoreCancelEvents" optional otherwise it will be a major change for everyone if you have to provide "false" for it.

**Comment by tb** [comment]
@fisshy I had similar issue, made PR with just scrollOffset, see https://github.com/fisshy/react-scroll/pull/225

**Pull request closed without merging**